src/Umbraco.Web.BackOffice/Extensions/UmbracoMembersUserServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using Umbraco.Core.Members;
+using Umbraco.Core.Security;
+using Umbraco.Core.Serialization;
+using Umbraco.Infrastructure.Members;
+using Umbraco.Web.BackOffice.Security;
+
+namespace Umbraco.Extensions
+{
+    public static class UmbracoMembersUserServiceCollectionExtensions
+    {
+        /// <summary>
+        /// Adds the services required for using Umbraco Members Identity
+        /// </summary>
+        /// <param name="services"></param>
+        public static void AddUmbracoMembersIdentity(this IServiceCollection services)
+        {
+            services.BuildUmbracoMembersIdentity()
+                .AddDefaultTokenProviders()
+                .AddUserStore<UmbracoMembersUserStore>()
+                .AddUserManager<IUmbracoMembersUserManager, UmbracoMembersUserManager>();
```

Comment:
**clarification:** could you point me to the implementation of this `AddUserManager` extension method?

Reply:
Found it. Silly Scott 🙄